.github/workflows/example_notebook_servers_publish.yaml

```diff
@@ -0,0 +1,107 @@
+name: Build & Publish Notebook Servers
+on:
+  push:
+    branches:
+      - master
+      - v*-branch
+    paths:
+      - components/example-notebook-servers/**
+      - releasing/version/VERSION
+
+jobs:
+  base_images:
+    name: Build & Push - Base
+    uses: ./.github/workflows/example_notebook_servers_publish_TEMPLATE.yaml
+    secrets: inherit
+    with:
+      build_arch: linux/amd64,linux/arm64
+      image_folders: |
+        base
+
+  codeserver_images:
+    name: Build & Push - Code Server
+    uses: ./.github/workflows/example_notebook_servers_publish_TEMPLATE.yaml
+    needs: [ base_images ]
+    secrets: inherit
+    with:
+      build_arch: linux/amd64,linux/arm64
```

Comment:
Have you also tried this build in GH runners? Am afraid that with trying to build both architectures at the same time we'll exhaust the resources, for which we ended up building serially in the other workflows
https://github.com/kubeflow/kubeflow/blob/master/.github/workflows/poddefaults_docker_publish.yaml#L46-L48

Reply:
@kimwnasptd yep, the workflows are designed very carefully to not exceed resource limits, and even when the build cache is empty, they still run successfully.

See the most recent few runs on my `thesuperzapper/kubeflow` repo: https://github.com/thesuperzapper/kubeflow/actions